src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -101,9 +54,11 @@ export const downloadAndUploadApprovedItemImageToS3 = async (
  */
 export const readImageFileFromDisk = (
   file: FileWithPath,
-  onloadCallBack?: VoidFunction
+  onloadCallBack?: VoidFunction,
+  fileReader?: FileReader
```

Comment:
would be nice to add a comment that this parameter exists for testing purposes. (at least i think that's the case, right?)